src/node/util.ts

```diff
@@ -206,8 +206,13 @@ export function deprecate(
   return fn;
 }
 
-export function getCallSite(frames: number = 10) {
-  return utilImpl.getCallSite(frames);
+// We leave this implementation for compat reasons.
+export function getCallSite(frames: number = 10): Record<string, string>[] {
+  return utilImpl.getCallSites(frames);
+}
```

Comment:
Perhaps instead...

```js
export const getCallSite = getCallSites;
```

So it's really just an alias.

Reply:
This requires changing the function signature to have a default value of 10, since C++ doesn't handle it right now. Doing it now. Just FYI.